core-tests/shared/src/test/scala/zio/stm/TMapSpec.scala

```diff
@@ -132,6 +132,26 @@ object TMapSpec extends ZIOBaseSpec {
       }
     ),
     suite("transformations")(
+      testM("toList") {
+        val elems = List("a" -> 1, "b" -> 2)
+        val tx =
+          for {
+            tmap <- TMap.make(elems: _*)
```

Comment:
You could use `TMap.fromIterable` here and below to clean things up slightly if you wanted to.

Reply:
updated